libs/dyn/convert/normalize.go

```diff
@@ -24,7 +24,7 @@ type normalizeOptions struct {
 	includeMissingFields bool
 }
 
-func Normalize(dst any, src dyn.Value, opts ...NormalizeOption) (dyn.Value, diag.Diagnostics) {
+func Normalize(dst any, src dyn.Value, field_substitutions map[string]string, opts ...NormalizeOption) (dyn.Value, diag.Diagnostics) {
```

Comment:
Please perform field substitution for TF in the same way we do for the other resources.

This can be reverted.

Reply:
Reverted